tests/models/dbrx/test_modeling_dbrx.py

```diff
@@ -30,7 +27,15 @@
     from transformers import DbrxForCausalLM, DbrxModel
 
 
-class DbrxModelTester:
+@require_torch
```

Comment:
We don't need such decorator(s) on `Tester` classes. I tried to remove it and everything is running ok.